config/install/image.style.anu_lms_beta_label.yml

```diff
@@ -0,0 +1,15 @@
+uuid: 606769ce-3641-45e4-8172-745025eb85cb
```

Comment:
uuid shouldn't be included in the install config. Just remove this line. As well as for all other config files in your commit.